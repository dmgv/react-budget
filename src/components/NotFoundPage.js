import React, { Fragment } from "react";
import { Link } from "react-router-dom";

const NotFoundPage = () => (
  <Fragment>
    <h1>Opsss! 404</h1>
    404 - <Link to="/">Go home</Link>
  </Fragment>
);

export default NotFoundPage;
