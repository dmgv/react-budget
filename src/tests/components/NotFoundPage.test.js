import React from "react";
import { shallow } from "enzyme";
import NotFoundPage from "../../components/NotFoundPage";

test("Should have a dashboard", () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(wrapper).toMatchSnapshot();
});
