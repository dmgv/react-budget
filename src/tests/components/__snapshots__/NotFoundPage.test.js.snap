// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should have a dashboard 1`] = `
<React.Fragment>
  <h1>
    Opsss! 404
  </h1>
  404 - 
  <Link
    replace={false}
    to="/"
  >
    Go home
  </Link>
</React.Fragment>
`;
